packages/Babylonian-Core.package/BPExampleToTest.class/instance/methodNotExistsOrCanBeOverritten.inClass..st

```diff
@@ -0,0 +1,11 @@
+private - serializing
+methodNotExistsOrCanBeOverritten: aMethodName inClass: aClass
```

Comment:
`isMethodNameApplicable: aMethodName in: aClass`, 'canMethodName: aMethodName beWrittenTo: aClass', 'isOverridable: aMethodName in: aClass' to remove comment